### DISCUSSION TITLE
Catalan language?

### DISCUSSION THREAD

**Discussion by mvilvi** [MM2-0/Kvaesitso#498] [General]
Hey everybody!
Is there any possibility to add the catalan language?
Thanks

**Comment by MM2-0** [comment]
Certainly! https://kvaesitso.mm20.de/docs/contributor-guide/i18n

**Comment by mvilvi** [comment]
Hi

I would like to be a catalan translator but i don't know how to do it. Cam you help me?

Marc
________________________________

**Comment by MM2-0** [comment]
1. Register at the Weblate instance, using your Github account: https://i18n.mm20.de/projects/kvaesitso/
2. (Add a new language; I have already done that for you)
3. Start translating: https://i18n.mm20.de/projects/kvaesitso/i18n/ca/

**Comment by mvilvi** [comment]
Many thanks!

Marc
________________________________

**Comment by mvilvi** [comment]
Hi,

All catalan language is done!!!

When we will see this language option added to your launcher?

Regards,
Marc

Missatge de Marc Vila ***@***.***> del dia ds., 19 d’ag. 2023 a les
21:13:

**Comment by MM2-0** [comment]
> All catalan language is done!!!

Thank you very much!

> When we will see this language option added to your launcher?

Whenever the next release is. I can't give a date yet, but I hope it's soon. Or, if you don't want to wait for that, it will be available in nightly builds starting tomorrow.
